src/Routers/UsersRouter.js

```diff
@@ -25,7 +26,20 @@ export class UsersRouter extends ClassesRouter {
     let data = deepcopy(req.body);
     req.body = data;
     req.params.className = '_User';
-    return super.handleCreate(req);
+
+    if (req.config.verifyUserEmails) {
```

Comment:
@nlutsenko wanted this logic in users controller instead. I think that makes sense.